src/dotnet/UI.Blazor/Components/TabPanel/tab-panel.css

```diff
@@ -63,40 +63,49 @@
     @apply max-w-full;
 }
 
-/* SideNav tabs */
-.side-nav .tab-panel-tabs {
+/* SideNav and Modal tabs */
+.side-nav .tab-panel-tabs,
```

Comment:
Can we use just '.tab-panel-tabs' without preceding selector?
So '.tab-panel-tabs' should have default styles. If we need to alter how it looks in specific component we should add modificators to css for that component.

Reply:
Done